src/main.tsx

```diff
@@ -1,11 +1,20 @@
 import "@rainbow-me/rainbowkit/styles.css";
```

Comment:
I think we can also remove it, produce an error on run

Reply:
Yes !! I don't see this CSS import thx :)